src/main/java/frc/robot/util/AutoPath.java

```diff
@@ -4,37 +4,38 @@
 import com.pathplanner.lib.PathPlanner;
 import com.pathplanner.lib.PathPlannerTrajectory;
 import edu.wpi.first.math.util.Units;
+import edu.wpi.first.wpilibj.DriverStation;
+import edu.wpi.first.wpilibj.DriverStation.Alliance;
 
 public class AutoPath {
   public static final double fieldLength = Units.inchesToMeters(651.25);
   public static final double fieldWidth = Units.inchesToMeters(315.5);
 
   public enum Autos {
-    GO_TO_FIRST_CARGO("goto1stcargo"),
+    GO_TO_FIRST_CARGO("dock"),
     GO_TO_GRID("gotogrid"),
     GO_TO_SECOND_CARGO("goto2ndcargo"),
     GO_TO_GRID_TWO("gotogrid2"),
     DOCK("dock"),
-    ONE_TO_A("grid1tocargoA"),
-    A_TO_THREE("cargoAtogrid3"),
-    THREE_TO_BRIDGE("getonthebridge");
-
-    private PathPlannerTrajectory trajectory;
+    ONE_TO_A("dock"),
+    A_TO_THREE("dock");
+    private PathPlannerTrajectory blueTrajectory, redTrajectory;
```

Comment:
what happened here lol

Reply:
the files did not contain valid paths so i just renamed them all to dock to prevent loading something invalid. it'll get cleaned up by your code